focus-android/app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteListFragment.kt

```diff
@@ -136,9 +136,7 @@ open class AutocompleteListFragment : BaseSettingsLikeFragment(), CoroutineScope
             adapter = DomainListAdapter()
             setHasFixedSize(true)
 
-            if (!isSelectionMode()) {
-                itemTouchHelper.attachToRecyclerView(this)
-            }
+            itemTouchHelper.attachToRecyclerView(this)
```

Comment:
This would fix the crash but regress the original specs and implementation from https://github.com/mozilla-mobile/focus-android/issues/1781 - https://github.com/mozilla-mobile/focus-android/commit/d1803f0ae7a7e8a18851ae52f5ddfbad66fc3d10. Dragging items should not be possible while in selection mode.